tests/test_dns_server_nonbreaking_changes.py

```diff
@@ -0,0 +1,213 @@
+# SPDX-FileCopyrightText: 2022 Martin Stephens
+#
+# SPDX-License-Identifier: MIT
+
+# pylint: disable=no-self-use, redefined-outer-name, protected-access, invalid-name, too-many-arguments
+"""Tests to confirm that there are no changes in behaviour to public methods and funtions."""
+import pytest
+
+from micropython import const
+import adafruit_wiznet5k.adafruit_wiznet5k_dns as wiz_dns
+from adafruit_wiznet5k.adafruit_wiznet5k_socket import socket
+
+#
+DEFAULT_DEBUG_ON = True
+
+
+@pytest.fixture
+def wiznet(mocker):
+    return mocker.patch("adafruit_wiznet5k.adafruit_wiznet5k.WIZNET5K", autospec=True)
+
+
+@pytest.fixture
+def wrench(mocker):
+    return mocker.patch(
+        "adafruit_wiznet5k.adafruit_wiznet5k_socket.socket", autospec=True
+    )
+
+
+class TestDNSInit:
+    def test_constants(self):
+        assert wiz_dns.QUERY_FLAG == const(0x00)
+        assert wiz_dns.OPCODE_STANDARD_QUERY == const(0x00)
+        assert wiz_dns.RECURSION_DESIRED_FLAG == 1 << 8
+
+        assert wiz_dns.TYPE_A == const(0x0001)
+        assert wiz_dns.CLASS_IN == const(0x0001)
+        assert wiz_dns.DATA_LEN == const(0x0004)
```

Comment:
A bunch of these constant accesses need to have a leading underscore on them now. Since these are brand new files those changes didn't come in when main was merged.

I'll add a commit to change them.

Reply:
I'll do it, I need to get all the Pytests tests passing.